deploy/chart/values.yaml

```diff
@@ -13,7 +13,10 @@ olm:
     ref: quay.io/operator-framework/olm:master
     pullPolicy: Always
   service:
-    internalPort: 8080
+    internalPort: 8443
+    externalPort: 8443
+  tlsSecret: olm-serving-cert
+  clientCASecret: pprof-serving-cert
```

Comment:
Shouldn't this remain HTTP by default? Users will need to customize this on their own based on the way they manage certificates.

Reply:
Isn't upstream built using the [deploy/upstream/values.yaml](https://github.com/operator-framework/operator-lifecycle-manager/blob/fffd2207fa0fac57bb99c59afe98823945c6b4b7/deploy/upstream/values.yaml) file? I agree with the sentiment behind your statement though.